### PR TITLE
Level2 unit: added test for Calculator and Webpage

### PR DESCRIPTION
**Pull request by TommyBacco**
6 tests for Calculator:

-  1 Case when no operator is specified (should return the input value)
-  4 Cases: 1 for every operation (add, sum, mul, div)
- 1 Case when a value is divided by 0 (should throw an error)

1 test for Webpage:

- Check if the request succeed and then _writeFile is called

### CODE REVIEW

File: content.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{"hello":"world"}
```
Comment (lurenzsgp) [inline content.txt]: Is this file required?

File: package.json [hunk 2]
```diff
@@ -24,7 +24,9 @@
   "dependencies": {
     "@types/node": "^12.12.11",
     "fs-extra": "^9.1.0",
-    "request": "^2.88.2"
+    "mock-http-server": "^1.4.2",
+    "request": "^2.88.2",
+    "yargs-parser": "^20.2.7"
```
Comment (lurenzsgp) [inline package.json]: For what are these package required?

File: src/Webpage.ts [hunk 3]
```diff
@@ -12,10 +12,12 @@ export class Webpage
      */
     public saveWebpage(url, path): Promise<string>
     {
-        return this.getWebpage(url).then((content) => this._writeFile(path, content)).catch((error) =>
+       // return Promise.resolve("true");
```
Comment (lurenzsgp) [inline src/Webpage.ts]: Remove commented lines

File: src/Webpage.ts [hunk 4]
```diff
@@ -12,10 +12,12 @@ export class Webpage
...
+
+         return this.getWebpage(url).then((content) => this._writeFile(path, content)).catch((error) =>
         {
             throw new Error(error);
-        })
+        })
```
Comment (lurenzsgp) [inline src/Webpage.ts]: 🙈 missing `;`

File: src/Webpage.ts [hunk 5]
```diff
@@ -12,10 +12,12 @@ export class Webpage
      */
     public saveWebpage(url, path): Promise<string>
     {
-        return this.getWebpage(url).then((content) => this._writeFile(path, content)).catch((error) =>
+       // return Promise.resolve("true");
+
+         return this.getWebpage(url).then((content) => this._writeFile(path, content)).catch((error) =>
```
Comment (lurenzsgp) [inline src/Webpage.ts]: 🙈remove ` ` at the start of the line

**Review by lurenzsgp — CHANGES_REQUESTED** [review]
Why have you removed the package-lock?

File: package.json (hunk 2 above)
Comment (TommyBacco) [reply package.json]: - mock-http-server create a mock server
- yargs-parser parse the json
- I removed request

Comment (lurenzsgp) [reply package.json]: Have you tried to use `JSON.parse()` to parse json object?

**Review by lurenzsgp — APPROVED** [review]
